### PR TITLE
feat: [CAL-1435] Emptyscreen for App Store if no external app has been configured (CALCOM-8154)

### LINKED ISSUE #8154: [CAL-1435] Emptyscreen for App Store if no external app has been configured
right now we only show Cal Video and instead we should show our EmptyScreen saying "No app has been configured yet. Please contact your admin"

CleanShot 2023-04-08 at 18 46 47@2x

<sub>[CAL-1435](https://linear.app/calcom/issue/CAL-1435/emptyscreen-for-app-store-if-no-external-app-has-been-configured)</sub>

### PR DESCRIPTION
**Pull request by gitstart-calcom**
Here is the GitStart Ticket for this pull request: [CALCOM-8154](https://developers.gitstart.com/client/calcom/ticket/CALCOM-8154)

As the empty screen is already implemented we added the string mentioned on description issue.

## Demo
https://www.loom.com/share/361a33211a80464193e9f67bb0134a3e

### CODE REVIEW

File: apps/web/components/apps/layouts/AppsLayout.tsx [hunk 1]
```diff
@@ -28,7 +28,7 @@ export default function AppsLayout({ children, actions, emptyStore, ...rest }: A
           {emptyStore ? (
             <EmptyScreen
               Icon={AlertCircle}
-              headline={t("no_apps")}
+              headline={isAdmin ? t("no_apps") : t("no_apps_contact_admin")}
```
Comment (gitstart-calcom) [inline apps/web/components/apps/layouts/AppsLayout.tsx:31]: Was added this conditional to show this string in this specefic case

File: apps/web/components/apps/layouts/AppsLayout.tsx [hunk 2]
```diff
@@ -28,7 +28,7 @@ export default function AppsLayout({ children, actions, emptyStore, ...rest }: A
...
               description={isAdmin ? "You can enable apps in the settings" : ""}
```
Comment (PeerRich) [inline apps/web/components/apps/layouts/AppsLayout.tsx:32]: lets i18n here, and also add a description for non-admins.

File: apps/web/public/static/locales/en/common.json [hunk 3]
```diff
@@ -1511,6 +1511,7 @@
   "no_available_apps": "There are no available apps",
   "no_available_apps_description": "Please ensure there are apps in your deployment under 'packages/app-store'",
   "no_apps": "There are no apps enabled in this instance of Cal",
+  "no_apps_contact_admin": "No app has been configured yet. Please contact your admin",
```
Comment (PeerRich) [inline apps/web/public/static/locales/en/common.json:1514]:
```suggestion
  "no_apps_contact_admin": "No app has been configured yet",
```


lets add "Please contact your admin" to the description instead

File: apps/web/components/apps/layouts/AppsLayout.tsx (hunk 2 above)
Comment (PeerRich) [inline apps/web/components/apps/layouts/AppsLayout.tsx:32]:
```suggestion
              description={isAdmin ? "You can enable apps in the settings" : "Please contact your admin"}
```

File: apps/web/components/apps/layouts/AppsLayout.tsx [hunk 4]
```diff
@@ -28,8 +28,8 @@ export default function AppsLayout({ children, actions, emptyStore, ...rest }: A
           {emptyStore ? (
             <EmptyScreen
               Icon={AlertCircle}
-              headline={t("no_apps")}
-              description={isAdmin ? "You can enable apps in the settings" : ""}
+              headline={isAdmin ? t("no_apps") : t("no_apps_contact_admin")}
+              description={isAdmin ? "You can enable apps in the settings" : "Please contact your admin"}
```
Comment (PeerRich) [inline apps/web/components/apps/layouts/AppsLayout.tsx:32]: lets extract i18n here

Comment (Udit-takkar) [reply apps/web/components/apps/layouts/AppsLayout.tsx:32]: @gitstart use i18n in description

**Review by Udit-takkar — APPROVED** [review]
code LGTM

**Pull request merged**